posthog/schema.py

```diff
@@ -428,6 +428,19 @@ class ConditionalFormattingRule(BaseModel):
     templateId: str
 
 
+class Math(StrEnum):
```

Comment:
This is a terrible name. It's called `CoreWebVitalsPercentile` in my code, so I'm not sure why is it that it's generating this terribly-named `Math` class